### PR TITLE
Add ManhwaFreak.fr and Sushiscan.fr

### LINKED ISSUE #16993: SushiScan.fr
### Source name

Sushiscan

### Source link

https://sushiscan.fr/

### Source language

French

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #17854: ManhwaFreak.Fr
### Source name

ManhwaFreak.Fr

### Source link

https://manhwafreak.fr/

### Source language

French

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by KirinRaikage**
Checklist:

Closes #17854 
Closes #16993 

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/overrides/mangathemesia/manhwafreakfr/src/ManhwaFreakFR.kt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.SChapter
+import okhttp3.Request
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ManhwaFreakFR : MangaThemesia("ManhwaFreak.fr", "https://manhwafreak.fr", "fr", dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale.ENGLISH)) {
```
Comment (alessandrojean) [inline multisrc/overrides/mangathemesia/manhwafreakfr/src/ManhwaFreakFR.kt:12]: Name it the same as their website calls itself?

```suggestion
class ManhwaFreakFR : MangaThemesia("ManhwaFreak", "https://manhwafreak.fr", "fr", dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale.ENGLISH)) {
```

File: multisrc/overrides/mangathemesia/sushiscanfr/src/SushiScanFR.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package eu.kanade.tachiyomi.extension.fr.sushiscanfr
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.source.model.SManga
+import org.jsoup.nodes.Document
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class SushiScanFR : MangaThemesia("Sushi-Scan.fr", "https://sushiscan.fr", "fr", dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale.FRENCH)) {
```
Comment (alessandrojean) [inline multisrc/overrides/mangathemesia/sushiscanfr/src/SushiScanFR.kt:9]: Same here.

```suggestion
class SushiScanFR : MangaThemesia("Sushiscan.fr", "https://sushiscan.fr", "fr", dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale.FRENCH)) {
```

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt [hunk 3]
```diff
@@ -79,6 +79,7 @@ class MangaThemesiaGenerator : ThemeSourceGenerator {
         SingleLang("MangKomik", "https://mangkomik.net", "id", overrideVersionCode = 1),
         SingleLang("Mangás Chan", "https://mangaschan.net", "pt-BR", className = "MangasChan", overrideVersionCode = 1),
         SingleLang("Manhwa Freak", "https://manhwa-freak.com", "en", overrideVersionCode = 2),
+        SingleLang("ManhwaFreak.fr", "https://manhwafreak.fr", "fr", className = "ManhwaFreakFR"),
```
Comment (alessandrojean) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt:82]:
```suggestion
        SingleLang("ManhwaFreak", "https://manhwafreak.fr", "fr", className = "ManhwaFreakFR"),
```

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt [hunk 4]
```diff
@@ -125,6 +126,7 @@ class MangaThemesiaGenerator : ThemeSourceGenerator {
         SingleLang("SummerToon", "https://summertoon.com", "tr"),
         SingleLang("Surya Scans", "https://suryascans.com", "en"),
         SingleLang("Sushi-Scan", "https://sushiscan.net", "fr", className = "SushiScan", overrideVersionCode = 6),
+        SingleLang("Sushi-Scan.fr", "https://sushiscan.fr", "fr", className = "SushiScanFR"),
```
Comment (alessandrojean) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt:129]:
```suggestion
        SingleLang("Sushiscan.fr", "https://sushiscan.fr", "fr", className = "SushiScanFR"),
```

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Comment by alessandrojean** [comment]
Still need to change the names in the generator.

**Comment by KirinRaikage** [comment]
Done.

**Pull request merged**